cmd/utils/flags.go

```diff
@@ -563,10 +568,18 @@ var (
 		Name:  "ping-ip-from-packet",
 		Usage: "Has the discovery protocol use the IP address given by a ping packet",
 	}
-	UseInMemoryDiscoverTable = cli.BoolFlag{
+	UseInMemoryDiscoverTableFlag = cli.BoolFlag{
```

Comment:
Rename the InMemoryDiscoveryTable flag to be more consistent with other flag arguments.